lib/active_merchant/billing/gateways/cyber_source.rb

```diff
@@ -862,11 +866,21 @@ def add_threeds_2_ucaf_data(xml, payment_method, options)
       def stored_credential_commerce_indicator(options)
         return unless options[:stored_credential]
 
-        return if options[:stored_credential][:initial_transaction]
+        if options[:stored_credential][:initial_transaction]
```

Comment:
I think it's best just to remove lines 869-875  since it doesn't work and we haven't heard back from Cybersource.